cmd/power_user.go

```diff
@@ -140,8 +144,8 @@ func powerUserExecWorker(userInput string) <-chan error {
 		s.Relationships = append(s.Relationships, mergeRelationships(relationships...)...)
 
 		bus.Publish(partybus.Event{
-			Type:  event.PresenterReady,
-			Value: syftjson.Format().Presenter(s),
+			Type:  event.SBOMReady,
```

Comment:
nit: pairing a function with something called "SBOMReady" makes it seem like there is an SBOM object that a subscriber would get, but instead gets a callback. Something like "Exit" doesn't imply receiving anything but doesn't exclude the possibility of a "finalizer" function.

Reply:
fair point. also, you mentioned the bus isn't exactly supposed to be an event system, so 👍